src/Game/UI/Gumps/OptionsGump.cs

```diff
@@ -4017,6 +4040,15 @@ private void Apply()
             _currentProfile.RelativeDragAndDropItems = _relativeDragAnDropItems.IsChecked;
             _currentProfile.HighlightContainerWhenSelected = _highlightContainersWhenMouseIsOver.IsChecked;
 
+            // TODO: Fix minimize button
```

Comment:
I need to remove this comment. Minimize is now functioning.

Reply:
> I need to remove this comment. Minimize is now functioning.

The minimize for certain gumps was done by me back then, only thing it lacks is a different placing for iconized vs non iconized GUMPS, but this is something that I cannot do without some internal mods (they are identified as same gump as original); at this time I don't see a real need for them, but this is a different story.

Anyway, why you want different styles of backpack universally? Isn't it more easyer to maintain your own mods without such a mainstream change? is this something present in original client?

Reply:
Hi SaschaKP, thank you for your review.

Yes, these additional backpack designs are featured as an option in the original client as of Publish 95 from Nov 17, 2016 (scroll to very bottom)
https://uo.com/wiki/ultima-online-wiki/publish-notes/publish-95/